catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -37,20 +37,26 @@
  * Validates an attribute's value(s) against a set of acceptable values.
  */
 public class EnumerationValidator implements AttributeValidator {
+    private final boolean ignoreCase;
+
     private final Set<String> values;
 
     /**
      * Constructs an {@code EnumerationValidator} with a given set of acceptable values.
      *
-     * @param values the values accepted by this validator
+     * @param ignoreCase whether enumeration validation should ignore case during evaluation
+     * @param values     the values accepted by this validator
      * @throws IllegalArgumentException if {@code values} is null or empty
      */
-    public EnumerationValidator(final Set<String> values) {
+    public EnumerationValidator(boolean ignoreCase, final Set<String> values) {
```

Comment:
Totally unimportant nit that i'll just leave here for posterity, Might make more sense to have 
`public EnumerationValidator(Set<String> values, boolean ignoreCase)` .. It just seems more conventional.
Maybe i'll tag this with my own tag and do it later
// todo (RCZ) - maybe swap these